.dependabot/config.yml

```diff
@@ -0,0 +1,18 @@
+# https://dependabot.com/docs/config-file/
+version: 1
+update_configs:
+  # Keep package.json (& lockfiles) up to date as soon as
+  # new versions are published to the npm registry
+  - package_manager: "javascript"
+    directory: "/"
+    update_schedule: "live"
```

Comment:
I think I'm gonna switch this to daily or weekly

Reply:
weekly seems fine